### DISCUSSION TITLE
Line item with numbers, hyphens are not working in my Gatsby blog site

### DISCUSSION THREAD

**Discussion by pajaydev** [gatsbyjs/gatsby#29365] [Help] [answered]
## Description
I love using Gatsby and I created my own blogging site using it. 
For some reason, Lists with numbers, bullet points & hyphens are not working in my Gatsby blog site. 
Thanks in advance 👍 

### Steps to reproduce
1. You can directly test it [here](https://ajayp.netlify.app/blog/2020/lasso).
2. Check the equivalent markdown [here](https://github.com/pajaydev/ajaykumar/blob/master/src/blogs/2020/lasso/index.md)

check out the [Source code](https://github.com/pajaydev/ajaykumar) to checkout and run in local.

### Expected result
It should have the numbers like this
<img width="1199" alt="Screen Shot 2021-02-05 at 6 55 53 PM" src="https://user-images.githubusercontent.com/21375014/107107086-cb3de380-67e3-11eb-85c2-f8aef689848b.png">

### Actual result

You can notice the empty space around it without numbers.
<img width="1081" alt="Screen Shot 2021-02-05 at 6 57 07 PM" src="https://user-images.githubusercontent.com/21375014/107107106-f4f70a80-67e3-11eb-9975-a9f5951b1fde.png">

### Environment
```
  System:
    OS: macOS 10.15.7
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 12.13.0 - /usr/local/bin/node
    Yarn: 1.22.10 - /usr/local/bin/yarn
    npm: 6.12.0 - /usr/local/bin/npm
  Languages:
    Python: 2.7.16 - /usr/bin/python
  Browsers:
    Chrome: 88.0.4324.146
    Firefox: 78.6.1
    Safari: 14.0.3
  npmPackages:
    gatsby: ^2.29.3 => 2.29.3 
    gatsby-plugin-google-analytics: ^2.8.0 => 2.8.0 
    gatsby-plugin-manifest: ^2.9.1 => 2.9.1 
    gatsby-plugin-react-helmet: ^3.3.4 => 3.3.4 
    gatsby-plugin-sass: ^2.8.0 => 2.8.0 
    gatsby-plugin-sharp: ^2.11.2 => 2.11.2 
    gatsby-plugin-webpack-bundle-analyzer: ^1.0.5 => 1.0.5 
    gatsby-remark-copy-linked-files: ^2.7.0 => 2.7.0 
    gatsby-remark-images: ^3.8.0 => 3.8.0 
    gatsby-remark-prismjs: ^3.10.0 => 3.10.0 
    gatsby-remark-smartypants: ^2.7.0 => 2.7.0 
    gatsby-source-filesystem: ^2.7.0 => 2.7.0 
    gatsby-transformer-remark: ^2.12.0 => 2.12.0 
  npmGlobalPackages:
    gatsby-cli: 2.19.0

```

**Comment by vladar** [comment, marked as answer]
That's because you have this [rule in `global.css`](https://github.com/pajaydev/ajaykumar/blob/9ae185a6e47cdc1e585cd5ada2f9d3094e6edd65/src/styles/global.css#L52-L54):

```css
li {
  list-style: none;
}
```

Remove it or add more specific rules for lists within your blog posts.

**Reply by pajaydev** [reply]
My bad, Thanks @vladar  for looking into it.
